internal/pkg/values/layer.go

```diff
@@ -244,47 +264,23 @@ func (l Layers) IsInGracePeriod(
 	return time.Now().Before(gracePeriodUntil), nil
 }
 
-// LayersToString gets a string representation of the layers with all fields printed explicitly.
-func (l Layers) LayersToString(layers []*Layer) []string {
-	result := make([]string, len(layers))
+// String gets the string representation of the layers.
+func (l Layers) String() string {
+	var builder strings.Builder
 
-	layersName := []string{"layerWorkload", "layerNamespace", "layerCli", "layerEnv"}
+	builder.WriteString("[")
 
-	for iterationNumber, layer := range layers {
-		excludeUntilStr := UndefinedString
-		if !layer.ExcludeUntil.IsZero() {
-			excludeUntilStr = layer.ExcludeUntil.String()
+	for iteration, item := range l {
+		if iteration > 0 {
+			builder.WriteString(" ")
 		}
 
-		result[iterationNumber] = fmt.Sprintf(
-			"{LayerName:%s, "+
-				"DownscalePeriod:%s, "+
-				"DownTime:%s, "+
-				"UpscalePeriod:%s, "+
-				"UpTime:%s, "+
-				"Exclude:%s, "+
-				"ExcludeUntil:%v, "+
-				"ForceUptime:%s, "+
-				"ForceDowntime:%s, "+
-				"DownscaleReplicas:%d, "+
-				"GracePeriod:%v, "+
-				"ExcludeNamespaces:%v, "+
-				"ExcludeDeployments:%v}",
-			layersName[iterationNumber],
-			layer.DownscalePeriod.String(),
-			layer.DownTime.String(),
-			layer.UpscalePeriod.String(),
-			layer.UpTime.String(),
-			layer.Exclude.String(),
-			excludeUntilStr,
-			layer.ForceUptime.String(),
-			layer.ForceDowntime.String(),
-			layer.DownscaleReplicas,
-			layer.GracePeriod,
-			layer.ExcludeNamespaces,
-			layer.ExcludeDeployments,
-		)
+		builder.WriteString(strconv.Itoa(int(LayerID(iteration))))
+		builder.WriteString(":")
+		builder.WriteString(fmt.Sprintf("%+v", item))
```

Comment:
I think if we are already using fmt.Sprintf we could just put the rest in there too. Also at least theoretically fmt.Fprintf(builder) is slightly more efficient.
```suggestion
		fmt.Fprintf(&builder, "%s:%+v", LayerId(i), item)
```